backend/integrations/discord/bot.py

```
@@ -36,12 +36,21 @@ def _register_queue_handlers(self):
         try:
+            # Sync globally
             synced = await self.tree.sync()
-            print(f"Synced {len(synced)} slash command(s)")
+            logger.info(f"Synced {len(synced)} global slash command(s)")
+
+            # Also sync to each guild for instant availability
+            for guild in self.guilds:
+                try:
+                    guild_synced = await self.tree.sync(guild=guild)
+                    logger.info(f"Synced {len(guild_synced)} commands to guild {guild.name}")
+                except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)